_posts/blog/2016-09-13-code-reviews.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Handbook Of Manners: How To Behave In A Code Review"
+description: Some thoughts on code reviews and the way they usually are and they should be performed.
+modified:
+categories: blog
+author: henning_gross
+excerpt: Code reviews are a common practise for knowledge distribution, learning and increasing quality - but do we perform them in a way that actually creates value?
+tags: [Mindset, Culture, Agile, XP]
+date: 2016-09-13T17:49:00-04:00
+---
+
+In most companies __code reviews__ are part of the __process__ today. In theory they help distributing knowledge, onboarding new people, increasing learning and quality.
+
+
+A code review, no matter if it is performed in a pairing manner or asynchronously is a situation in which one person __inspects__ the work of another person.
+
+
+This can lead to the __assumption__ that the reviewer has to accept the changes the reviewee wants to introduce and put the reviewer in a position of power over the reviewee who has to __seek for approval__.
```

Comment:
Ok, I changed from reviewee to author. Maybe this is not the right time to invent new words. Is it?